src/layouts/AppLayout.tsx

```diff
@@ -326,7 +326,7 @@ export default function AppLayout({
                           : "text-gray-100 hover:bg-secondary-lighter",
                         "group flex items-center px-2 py-2 text-base font-medium rounded-md focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-secondary-dark focus:ring-secondary-lighter"
                       )}
-                      onClick={() => setSidebarOpen(false)}
+                      onClick={() => (window.location.href = item.href)}
```

Comment:
This should be a react router link, so we don't do a hard refresh on page navigation. The original `onClick` closed the sidebar on mobile. So, please make sure we test everything on mobile, as well.